CHANGELOG.md

```diff
@@ -35,6 +35,21 @@
 
 ### Fixes
 
+- Fix high percentage of slow frames (#1915)
+## Unreleased
+
+### Features
+
+- Upload frame rendering timestamps to correlate to sampled backtraces (#1910)
+
+### Fixes
+
+- Remove WebKit optimization check (#1921)
+
+## 7.18.1
+
+### Fixes
```

Comment:
`m`: I'm not sure that this change belongs here. Maybe also worth checking the changelog entry above.

Reply:
man, i don't know what's going on with nerves to the changelog, will fix this again. I've been adding it under features.